.coafile

```diff
@@ -22,3 +22,8 @@ bears = MarkdownBear, AlexBear
 
 [Git]
 bears = GitCommitBear
+
+[YAML]
+files = **.yml
```

Comment:
so are all yml files in this repo grabbed with that glob? I just want to confirm that you've checked.
(👉 https://github.com/coala/coala-atom/pull/79#discussion_r151466981)